admin_app_2/src/components/ContentModal.tsx

```diff
@@ -0,0 +1,140 @@
+import React from "react";
+import { Modal, Typography, Box, Fade, Button, Chip } from "@mui/material";
+import { appColors, appStyles, sizes } from "@/utils";
+import { Layout } from "./Layout";
+import {
+  Close,
+  Delete,
+  Edit,
+  RemoveRedEye,
+  ThumbDown,
+  ThumbUp,
+} from "@mui/icons-material";
+import LanguageButtonBar from "./LanguageButtonBar";
+import Link from "next/link";
+import Image from "next/image";
+
+const ContentViewModal = ({
+  open,
+  onClose,
+}: {
+  open: boolean;
+  onClose: () => void;
+}) => {
+  return (
+    <Modal
+      open={open as boolean}
+      onClose={onClose}
+      sx={[
+        { display: "flex" },
+        appStyles.alignItemsCenter,
+        appStyles.justifyContentCenter,
+      ]}
+    >
+      <Fade in={!!open}>
+        <Box
+          sx={{
+            width: "70%",
+            height: "70%",
+            backgroundColor: appColors.white,
+            p: sizes.doubleBaseGap,
+          }}
+        >
+          <Layout.FlexBox
+            flexDirection={"row"}
+            {...appStyles.justifyContentSpaceBetween}
+          >
+            <Typography variant="h5">Content #142 </Typography>
+            <Close onClick={onClose} />
+          </Layout.FlexBox>
+
+          <Layout.Spacer multiplier={1} />
+          {["Pollution", "COVID-19", "Masks"].map((topic, index) => (
+            <Chip
+              key={index}
+              label={topic}
+              sx={{ mx: sizes.tinyGap }}
+              disabled
+            />
+          ))}
+          <Layout.Spacer multiplier={1} />
+
+          <Layout.FlexBox
+            flex={1}
+            flexDirection={"row"}
+            sx={{
+              height: "85%",
+            }}
+          >
+            <Layout.FlexBox
+              flex={0.7}
+              flexDirection={"column"}
+              sx={{ backgroundColor: appColors.white }}
+            >
+              <LanguageButtonBar expandable={false} />
```

Comment:
These flexboxes are not working properly (resize the window to see). It seems that the issue if from the flexDirection - things look better horizontally if I comment that out, but break with vertical resizing...